README.md

```diff
@@ -33,10 +33,19 @@ To fetch a timestamp with the provided `timestamp-cli`:
 To fetch a timestamp with `openssl` and `curl`:
 
 1. Retrieve the verification chain: `curl http://localhost:3000/api/v1/timestamp/certchain > ts_chain.pem`
+1. Split chain into root CA certificate and "untrusted" intermediate and leaf certificates:
+   1. Split: `csplit -s -f tmpcert- ts_chain.pem '/-----BEGIN CERTIFICATE-----/' '{*}'`
```

Comment:
I tried running this `csplit` command but got the message `csplit: *}: bad repetition count`

Reply:
Not supported on macOS, so I added a brew command to install the GNU version and which utility to use.